validator/src/admin_rpc_service.rs

```diff
@@ -815,8 +815,12 @@ pub async fn connect(ledger_path: &Path) -> std::result::Result<gen_client::Clie
     }
 }
 
-pub fn runtime() -> jsonrpc_server_utils::tokio::runtime::Runtime {
-    jsonrpc_server_utils::tokio::runtime::Runtime::new().expect("new tokio runtime")
+pub fn runtime() -> Runtime {
+    tokio::runtime::Builder::new_multi_thread()
+        .thread_name("solAdminRpcRt")
+        .enable_all()
+        .build()
+        .unwrap()
```

Comment:
Keep the expect?

Reply:
Similar note as other comment about the `.unwrap()` vs. `.expect()` here; I don't think the `.expect()` does us much (the backtrace will be more informative than the message) and a better result would be passing an error and letting caller handle the error properly.

Looking at the callers of this `runtime()` function, it looks like there would be a bit more of a refactor to properly handle error. Maybe I can do it while I'm collecting data for thread pool size reductions later on